app/src/common/components/DeleteDialog.tsx

```diff
@@ -0,0 +1,95 @@
+import React from "react";
+
+import {
+  Button,
+  Dialog,
+  DialogProps as MuiDialogProps,
+  DialogTitle,
+  DialogContent,
+  DialogContentText,
+  DialogActions,
+  makeStyles,
+  Typography,
+  CircularProgress,
+} from "@material-ui/core";
+import clsx from "clsx";
+import { useTranslation } from "react-i18next";
+
+type DeleteDialogProps = MuiDialogProps & {
+  title: string;
+  loading?: boolean;
```

Comment:
why is that optional?

Reply:
I suggested it because I prefer props to be optional when it's not a big deal in the code